sys/net/application_layer/sock_dns/dns.c

```diff
@@ -29,6 +32,40 @@
 /* global DNS server UDP endpoint */
 sock_udp_ep_t sock_dns_server;
 
+#ifdef MODULE_AUTO_INIT_SOCK_DNS
+void auto_init_sock_dns(void)
+{
+    assert(CONFIG_AUTO_INIT_SOCK_DNS_IP_VERSION == 4
+        || CONFIG_AUTO_INIT_SOCK_DNS_IP_VERSION == 6);
+
+    assert(CONFIG_AUTO_INIT_SOCK_DNS_SERVER_PORT > 0
+        && CONFIG_AUTO_INIT_SOCK_DNS_SERVER_PORT <= 0xffff);
+
+    switch(CONFIG_AUTO_INIT_SOCK_DNS_IP_VERSION) {
```

Comment:
Can you uncrustify? That should add missing spaces and potentially others..

Reply:
I ran `uncrustify` and just realized that not indenting switch labels is a coding convention in RIOT these days. I can't say I'm a friend of this rule, but as it is convention...